src/global/global_init.cc

```diff
@@ -193,6 +193,7 @@ global_init(const std::map<std::string,std::string> *defaults,
 
   if (g_conf->fatal_signal_handlers)
     install_standard_sighandlers();
+  register_assert_context(g_ceph_context);
```

Comment:
Previous if doesn't have brackets  which is not very obvious at a quick read. The register call looks badly indented while its not. Maybe adding brackets on the if or keeping an empty line before it could avoid this visual confusion.

Reply:
fixed